marketplace-core/pom.xml

```diff
@@ -97,6 +97,10 @@
       <groupId>org.webjars.bower</groupId>
       <artifactId>angular-ui-bootstrap-bower</artifactId>
     </dependency>
+    <dependency>
```

Comment:
Is our own client code really better than the 3rd party httpclient code (more secure) or is it simply assumed to be so since it isn't flagged as a risk?
 
Generally agree, but a new project...sigh. Also hate the thought of yet another kettle dependency. Especially if it's a generic thing that is not specific to kettle.

Maybe there is a nice 3rd party http client class that is not so insecure? Some research here might save us pain (maybe it's already been done).